docs/modules/ROOT/pages/reference/api-reference/neo4jgraphql.adoc

```diff
@@ -29,162 +29,10 @@ Accepts all of the options from https://www.graphql-tools.com/docs/generate-sche
  Type: https://neo4j.com/docs/javascript-manual/current/[`Driver`]
 |An instance of a Neo4j driver.
 
-|`config` +
- +
- Type: xref::reference/api-reference/neo4jgraphql.adoc#api-reference-neo4jgraphql-input-neo4jgraphqlconfig[`Neo4jGraphQLConfig`]
-|Additional Neo4j GraphQL configuration options.
-
 |`features` +
  +
  Type: xref::reference/api-reference/neo4jgraphql.adoc#api-reference-neo4jgraphql-input-neo4jfeaturessettings[`Neo4jFeaturesSettings`]
 |Could be used for configure/enable/disable specific features. 
```

Comment:
Does `validate` need to be added here if we're not completely getting rid of this for now?

Reply:
I am going to completely get rid of it, hence why I haven't added new stuff. Will probably remove when it's in the dedicated docs repo. 🙂